samples/nrf_rpc/protocols_serialization/client/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -7,28 +7,34 @@
 	chosen {
 		nordic,rpc-uart = &uart21;
 	};
+
+	/* delete all buttons except button0 to free GPIO pins assigned to uart21 below */
+	buttons {
+		/delete-node/ button1;
+		/delete-node/ button2;
+		/delete-node/ button3;
+	};
 };
 
 &pinctrl {
 	uart21_default: uart21_default {
 		group1 {
-			psels = <NRF_PSEL(UART_TX, 1, 11)>,
-				<NRF_PSEL(UART_RX, 1, 10)>,
-				<NRF_PSEL(UART_RTS, 1, 12)>,
-				<NRF_PSEL(UART_CTS, 1, 13)>;
+			psels = <NRF_PSEL(UART_TX, 1, 8)>,
```

Comment:
Did you try to enable pulls-up? Is the issue with lost bytes back after this?

Reply:
For me pull-up never caused any issues, so I'll let you test if they should be brought back :)

Reply:
That is very strange since pins connected to LED have resistors allowing appropriate current for LED. Then connecting in parallel pull-up resistor will make common resisting less. Rled*Rpullup / (Rled + Rpullup). Unfortunately, I didn't find anywhere electrical schematic of nrf54l15 dk to confirm it.

Reply:
Found it. The situation is even worse :)
![Screenshot from 2024-11-07 10-36-20](https://github.com/user-attachments/assets/848aa1dc-e9e8-4eb8-b559-e002f1c89c87)

Reply:
:) so you verified that restoring pull-ups now works for you?

Reply:
ok, restored pull-ups. Works for me at least.

Reply:
I checked with analyzer. It doesn't seem that legacy shim uses CTS\RTS in the same manner as the newest one. 
![Screenshot from 2024-11-07 13-36-46](https://github.com/user-attachments/assets/890ecf2e-253a-4a5a-988c-7c9f9b6a44e4)
Usage pulls up doesn't impact on it.

Reply:
btw. checked it one more time after adding pull up resistor. I see good signal on CTS\RTS line. No nanoseconds pikes anymore. Seems it works.